<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculadora</class>
 <widget class="QMainWindow" name="Calculadora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>420</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Calculadora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="display">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>107</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn8">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>107</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn9">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>107</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>184</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>184</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn6">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>184</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>261</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>261</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>261</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_porcent">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>338</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn0">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>338</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_virgula">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>338</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_soma">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>107</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>107</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_subtracao">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>184</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>184</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multiplicacao">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>261</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_igual">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>260</y>
      <width>65</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>140</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>140</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_divisao">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>338</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
